src/state-summary/state-card-content.html

```diff
@@ -42,7 +42,11 @@
   },
 
   _maybeLoadCustomUi: function (stateType) {
-    var isLoaded = this._isLoaded('STATE-CARD-' + stateType.toUpperCase());
+    // Constructor check doesn't work on Safari 10.
+    // Bypassing this check mean that in order to user a built-in
```

Comment:
We should not allow people to use built-in components as custom ui. We should remove that functionality completely.